### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by AbhishekPSI7042** [PaddlePaddle/PaddleOCR#12663] [Q&A]
SystemError: (Fatal) Blocking queue is killed because the data reader raises an exception.
  [Hint: Expected killed_ != true, but received killed_:1 == true:1.] (at ../paddle/fluid/operators/reader/blocking_queue.h:175)

**Comment by phamkhactu** [comment]
Have you solve it?

**Comment by SWHL** [comment]
I guess the reason maybe caused by PaddlePaddle. Please try to check by running the following:
```python
import paddle

paddle.utils.run_check()
```
